modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -53,6 +53,7 @@
             <class name="org.wso2.am.integration.tests.other.APIM678ApplicationCreationTestCase" />
             <class name="org.wso2.am.integration.tests.other.APIM710AllSubscriptionsByApplicationTestCase"/>
             <class name="org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase"/>
+            <class name="org.wso2.am.integration.tests.other.APIM714GetAllDocumentationTestCase"/>
```

Comment:
Lets include the testng at once at the end. We can remove this for now.

Reply:
Fixed